Makefile

```diff
@@ -418,12 +426,6 @@ define testapp-set-replicas
 	kubectl scale deployment frontend-podinfo -n test-gslb --replicas=$1
 endef
 
-define hit-testapp-host
-	kubectl run -it --rm busybox --restart=Never --image=busybox -- sh -c \
-	"echo 'nameserver `$(K8GB_COREDNS_IP)`' > /etc/resolv.conf && \
-	wget -qO - $1"
-endef
```

Comment:
Could you please clarify this removal?

Reply:
@ytsarev Sure, this is the removal of the duplicate definition to https://github.com/k8gb-io/k8gb/pull/495/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L403 I've noticed while updating the Makefile.

Reply:
I can make this fix as separate PR to split the scope.

Reply:
@ytsarev will be done separately, amended the changes.